frontend/src/components/DetailDrawer.css

```
@@ -1622,22 +1624,22 @@ a.drawer-related-news-title:focus-visible {
 }
 
 .refs-link:hover {
-  color: #9ecbff;
+  color: var(--text-heading);
   text-decoration-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)